sdk/resourcemanager/Azure.ResourceManager/src/Shared/ManagedServiceIdentityTypeV3Converter.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Azure.ResourceManager.Models
+{
+    /// <summary> JsonConverter for managed service identity type v3. </summary>
+    internal partial class ManagedServiceIdentityTypeV3Converter : JsonConverter<ManagedServiceIdentityType>
+    {
+        private const string _systemAssignedUserAssignedV3Value = "SystemAssigned,UserAssigned";
+
+        /// <summary> JsonConverter for managed service identity type v3. </summary>
+        public static ManagedServiceIdentityTypeV3Converter Default
```

Comment:
Can we just have the users new the object instead of providing this default?

Reply:
Benchmark tests showed similar results for the two (with creating the JsonConverter 10 times). I will change to use the new keyword to make it more clear:
|               Method |     Mean |     Error |    StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|--------------------- |---------:|----------:|----------:|-------:|------:|------:|----------:|
|  UseDefaultBenchmark | 6.196 us | 0.1225 us | 0.2081 us | 0.8087 |     - |     - |  23.43 KB |
| NewInstanceBenchmark | 6.085 us | 0.1211 us | 0.2781 us | 0.7706 |     - |     - |  23.66 KB |